photometry.py

```diff
@@ -17,6 +17,13 @@
 sex = 'sextractor'
 fields_core_path = conf.core_path+'/source/sdss/dr12/plates/'
 
+# Directories containing default files for SExtractor:
+SExtractor_default_files_path = ''
```

Comment:
Great ! :+1:
Now the best would be to provide a default folder with these files, and put its path here

Reply:
Hm you are right. So lets put them into the tarball. and add the path here.

Reply:
It's just a few Kbytes right ? No need for a tarball, just include the folder in the repository

Reply:
Ok!